tools/analysistools.py

```diff
@@ -491,4 +491,68 @@ def plotScanCustom(scanData, scanLabel, scanIndices, interest, indexOffset, aggr
     if dump:
         plt.savefig('{}.png'.format(plotName),dpi=dpi);
     if visual:
-        plt.show();
\ No newline at end of file
+        plt.show();
+
+def measureLigandContact(xyzaFile, silent=True):
+    headersize = 9
+    xyzallsize = 2973
```

Comment:
Why is this not calculated based on the length of the file, after all we do allow the timestep and runtime to change?